partner_priority/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds priority levels on partners.
```

Comment:
Could you improve the module description?

In wich case is necessary this module?

Reply:
I added some more to the description, my company will be using this to set a 'sensitivity' level for our customers. Originally I named this model as 'Partner Sensitivity' however priority used throughout Odoo so for consistency, I renamed it to 'Partner Priority' so others can use it for their own business needs.